docs/source/getstart.rst

```diff
@@ -0,0 +1,227 @@
+
+          if train_iter.is_new_epoch:
+
+              # [ChainerUI] write values to 'log' file
+              stats = {
+                  'epoch': train_iter.epoch,
+                  'iteration': train_iter.epoch * args.batchsize,
+                  'train/loss': train_loss, 'train/accuracy': train_accuracy,
+                  'test/loss': test_loss, 'test/accuracy': test_accuracy
+                  }
+              ui_report(stats)
+
+.. note::
+
+   :ref:`chainerui's LogReport <module_log_report>` only sets ``exampled_time``, so train loop has to set ``epoch`` and ``iteration`` manually.
+
+Experimental conditions
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: ../images/result_table_condition_sample.png
+
+ChainerUI shows training job with experimental conditions read from ``args`` file. ``args`` file is a JSON file, which includes key-value pairs. See :ref:`save_args <module_save_args>`, util function to dump command line arguments or dictionary to ``args`` file.
+
+Setup example, a brief extract taken from `MNIST example <https://github.com/chainer/chainerui/blob/master/examples/train_mnist.py>`__:
+
+.. code-block:: python
+
+  # [ChainerUI] import chainerui util function
+  from chainerui.utils import save_args
+
+  def main():
+      parser.add_argument('--out', '-o', default='result',
+                          help='Directory to output the result')
+      args = parser.parse_args()
+
+      # [ChainerUI] save 'args' to show experimental conditions
+      save_args(args, args.out)
+
+``args`` file example, values are showed as experimental conditions on a results table::
+
+  {
+      "resume": "",
+      "batchsize": 100,
+      "epoch": 20,
+      "frequency": -1,
+      "gpu": 0,
+      "unit": 1000,
+      "out": "results"
+  }
+
+Operate training loop
+~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: ../images/detail_page_operation_block.png
+
+ChainerUI support to operate a training loop with :ref:`CommandsExtension <module_command_extension>`, the latest version supports:
+
+* take a snapshot
+* adjust hyperparameters of an optimizer
+
+Operation buttons are in :ref:`detail page <ui_detail_page>`.
+
+Setup example, a brief extract taken from `MNIST example <https://github.com/chainer/chainerui/blob/master/examples/train_mnist.py>`__:
+
+.. code-block:: python
+
+  from chainer import training
+  from chainer.training import extensions
+
+  # [ChainerUI] import CommandsExtension
+  from chainerui.extensions import CommandsExtension
+
+  def main():
+      trainer = training.Trainer(updater, (args.epoch, 'epoch'), out=args.out)
+
+      # [ChainerUI] Observe learning rate
+      trainer.extend(extensions.observe_lr())
+      # [ChainerUI] enable to send commands from ChainerUI
+      trainer.extend(CommandsExtension())
+
+.. note::
+
+   This operation to training loop is given by :ref:`CommandsExtension <module_command_extension>` which is on the premise of ``Trainer``. A training loop without ``Trainer`` cannot use this function.
```

Comment:
This operation of a training loop is from the CommandsExtension, which requires Trainer...